Sources/MongoSwift/MongoSwift.swift

```diff
@@ -1,4 +1,40 @@
+import libmongoc
+
 /// Returns a `String` indicating the version of the driver.
 public func version() -> String {
     return "testing"
 }
+
+/// A utility class for libmongoc initialization and cleanup.
+final public class MongoSwift {
+    final class MongocInitializer {
+        static let shared = MongocInitializer()
+
+        private init() {
```

Comment:
A private initializer was advised in [this write-up](https://krakendev.io/blog/the-right-way-to-write-a-singleton). That said, I'm not sure if MongocInitializer is accessible from outside of the MongoSwift class, so maybe that isn't a concern?

Reply:
yeah I think `private` is indeed the way to go here